src/Libraries/CoreNodes/Math.cs

```diff
@@ -231,7 +231,7 @@ public static long Abs(long integer)
         ///     Finds the inverse cosine, the angle whose cosine is the given ratio.
         /// </summary>
         /// <param name="ratio">The cosine of the angle, a number in the range [-1, 1].</param>
-        /// <returns name="angle">The angle whose cosine is the input ratio.</returns>
+        /// <returns name="double">The angle whose cosine is the input ratio. (double)</returns>
```

Comment:
`(double)` seems unnecessary. It already says double in the name.

Reply:
adressed in 
0cb98d5